hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/token/UniqueTokenManagementSpecs.java

```diff
@@ -813,28 +815,31 @@ private HapiSpec mintUniqueTokenReceiptCheck() {
                                 .initialSupply(0)
                                 .supplyKey(SUPPLY_KEY)
                                 .treasury(TOKEN_TREASURY))
-                .when(mintToken(A_TOKEN, List.of(metadata("memo"))).via(MINT_TRANSFER_TXN))
+                .when(mintToken(A_TOKEN, List.of(metadata("memo"))).via(mintTransferTxn))
                 .then(
                         UtilVerbs.withOpContext((spec, opLog) -> {
-                            var mintNft = getTxnRecord(MINT_TRANSFER_TXN);
+                            var mintNft = getTxnRecord(mintTransferTxn);
                             allRunFor(spec, mintNft);
                             var tokenTransferLists = mintNft.getResponseRecord().getTokenTransferListsList();
                             Assertions.assertEquals(1, tokenTransferLists.size());
-                            tokenTransferLists.stream().forEach(tokenTransferList -> {
+                            tokenTransferLists.forEach(tokenTransferList -> {
                                 Assertions.assertEquals(
                                         1,
                                         tokenTransferList.getNftTransfersList().size());
-                                tokenTransferList.getNftTransfersList().stream().forEach(nftTransfers -> {
+                                tokenTransferList.getNftTransfersList().forEach(nftTransfers -> {
                                     Assertions.assertEquals(
-                                            AccountID.getDefaultInstance(), nftTransfers.getSenderAccountID());
+                                            AccountID.newBuilder()
+                                                    .setAccountNum(0)
+                                                    .build(),
+                                            nftTransfers.getSenderAccountID());
```

Comment:
Both mono and mod code fail when this assertion checks for `AccountID.getDefaultInstance()`, and both pass when we change to account 0.0.0. Not sure how this is passing in CI checks on mono service 😬

Reply:
👍

Reply:
> Not sure how this is passing in CI checks on mono service 😬

Easy---it's not running in CI! 😝